ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -103,6 +106,14 @@
   private static final String KEYBOARD_TYPE_NUMBER_PAD = "number-pad";
   private static final String KEYBOARD_TYPE_PHONE_PAD = "phone-pad";
   private static final String KEYBOARD_TYPE_VISIBLE_PASSWORD = "visible-password";
+  private static final List<String> KEYBOARD_TYPES = new ArrayList<>(Arrays.asList(
```

Comment:
Can we use a Set instead, so that the check on line 731 is done in a constant time?